tests/test_cte.py

```diff
@@ -19,6 +19,7 @@ def test_with_select_one_without_as():
     assert_table_lineage_equal(
         "WITH wtab1 (SELECT * FROM schema1.tab1) SELECT * FROM wtab1",
         {"schema1.tab1"},
+        dialect="mysql",
```

Comment:
I believe this is a bug from sqlfluff. This syntax is actually sparksql only. MySQL will throw syntax error.

Reply:
I think for this case, we can open a PR in `sqlfluff`.

Reply:
For the moment, I have removed mysql from this test case despite it works. Only tested against databricks and sparksql because this is documented as a feature in their language manuals.